packages/types/src/types.ts

```diff
@@ -186,7 +186,11 @@ export type IgnoreConfig = {
   assistants: {
     [assistantName: string]: {
       rules: {
-        [ruleName: string]: { allObjects: true } | { objects: string[] }
+        [ruleName: string]:
+          | { allObjects: true } // Rule full ignored
+          | { allObjects: true; objects: [] } // Rule full ignored
```

Comment:
Can we include some context in the comment why both `allObjects` and `objects` may be present? If I understand #82 correctly it's about setting `allObjects` to true when specific objects were ignored before so you don't have to specifically remove them 🤔

Reply:
Not much has changed here - both permutations of the type mean that the rule is full ignored, whether `objects` are present or not. In other words, the `objects` array is ignored, if `allObjects` is present. I'm just more accurately modelling the data that @craig006 is passing in - I learned he retains the contents of `objects` if `allObjects` is toggled on, so that if a user toggles `allObjects` back off, their previously ignored objects state is still present.

Reply:
Added a more descriptive comment.